variables.tf

```diff
@@ -15,6 +15,12 @@ variable "api_servers_external" {
   default     = []
 }
 
+variable "api_servers_ips" {
+  description = "List of public kube-apiserver IPs to include in the TLS certificate"
```

Comment:
Do the IPs have to be public?

Reply:
Maybe "external" would be a better word?

Reply:
Could maybe also be used for internal access to the API? I can delete the "public" in this description.

Reply:
I suggest we leave things as generic as possible. AFAICT any IP address would work here, so no need to restrict things.

Reply:
@pothos I **would** consider mentioning something more specific in the module which calls terraform-render-bootkube though.